lib/oxide-vpc/src/api.rs

```diff
@@ -899,6 +939,32 @@ impl Display for Ports {
     }
 }
 
+/// Add an entry to the gateway allowing a port to send or receive
+/// traffic on a CIDR other than its private IP.
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct AllowCidrReq {
+    pub port_name: String,
+    pub cidr: IpCidr,
+    pub dir: Direction,
+}
+
+/// Remove entries from the gateway allowing a port to send or receive
+/// traffic on a specific CIDR other than its private IP.
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct RemoveCidrReq {
+    pub port_name: String,
+    pub cidr: IpCidr,
+    pub dir: Direction,
+}
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub enum RemoveCidrResp {
```

Comment:
this seems so option-like, so maybe it's worth keeping around info in the Ok case (maybe for info tracking) on what was removed?

Reply:
Sure, that's a good idea. Done via 527d6ff.